bluebrain/repo-bluebrain/packages/py-astrovascpy/package.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAstrovascpy(PythonPackage):
+    """
+    Vasculature blood flow computation and impact of astrocytic
+    endfeet on vessels
+    """
+
+    homepage = "https://github.com/BlueBrain/AstroVascPy"
+
+    url = "https://github.com/BlueBrain/AstroVascPy/releases/tag/0.1"
+
+    maintainers("tristan0x")
+
+    version("0.1", sha256="1e9b2036e9c1ce4cb0650612c0be2e7e7b142754e1550dfbfb0b757cee4f8bdd")
+
+    variant("vtk", default=False, description="add VTK support (mainly for visualization)")
+
+    depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-setuptools-scm", type=("build", "run"))
+
+    depends_on("py-click", type=("build", "run"))
+    depends_on("py-cached-property", type=("build", "run"))
+    depends_on("py-coverage", type=("build", "run"))
+    depends_on("py-cython", type=("build", "run"))
+    depends_on("py-h5py", type=("build", "run"))
+    depends_on("py-libsonata", type=("build", "run"))
+    depends_on("py-matplotlib", type=("build", "run"))
+    depends_on("py-morphio", type=("build", "run"))
+    depends_on("py-networkx", type=("build", "run"))
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-pandas", type=("build", "run"))
+    depends_on("petsc", type=("build"))
```

Comment:
```suggestion
    depends_on("petsc")
```
`petsc` does not look like a header-only library.

Reply:
fixed in [6394ff3](https://github.com/BlueBrain/spack/pull/2175/commits/6394ff3ea1348e15563b27ed93c354f9f96e687a)